browsers/opera_android.json

```
@@ -51,16 +52,19 @@
           "engine_version": "26"
         },
         "15": {
+          "release_date": "2013-07-02",
           "status": "retired",
           "engine": "Blink",
           "engine_version": "28"
         },
         "16": {
+          "release_date": "2013-08-26",
           "status": "retired",
           "engine": "Blink",
           "engine_version": "29"
         },
         "18": {
+          "release_date": "2013-09-13",
```

This looks like the date for a "developer" release of version 18 (maybe a nightly channel?)
https://github.com/operasoftware/devopera/blob/master/src/blog/_posts/2013-09-13-opera-18-developer-and-opera-16-beta-for-android.md

Looks like the final release was on November 20: https://blogs.opera.com/news/2013/11/opera-18-android-tablet/

```suggestion
          "release_date": "2013-11-20",
          "release_notes": "https://blogs.opera.com/news/2013/11/opera-18-android-tablet/",
```